dist/fast-json-patch.js

```diff
@@ -82,6 +82,11 @@ var __extends = (this && this.__extends) || function (d, b) {
  * (c) 2017 Joachim Wester
  * MIT license
  */
+var _hasOwnProperty = Object.prototype.hasOwnProperty;
+function hasOwnProperty(obj, key) {
+    return _hasOwnProperty.call(obj, key);
```

Comment:
Can we somehow avoid the `.call` here? I am afraid that we can't because we have to pass `obj` as `this` argument to `Object.prototype.hasOwnProperty` but perhaps I am missing something here.